MultiServer.py

```diff
@@ -1235,6 +1235,7 @@ def __new__(cls, name, bases, attrs):
             commands.update(base.commands)
         commands.update({command_name[5:]: method for command_name, method in attrs.items() if
                          command_name.startswith("_cmd_")})
+        attrs["aliases"] = {}
```

Comment:
maybe rename to "command_aliases" to not make it confusing with player name aliases